editor/src/components/tpc-templates-button.js

```diff
@@ -0,0 +1,25 @@
+import { __ } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+import { useState } from '@wordpress/element';
+import ImportModal from './import-modal';
+import { iconBlack as icon } from '../icon';
+
+const TpcTemplatesButton = () => {
+	const modalState = useState( false );
+
+	return (
+		<>
+			<Button
+				icon={ icon }
+				isPrimary
+				onClick={ () => modalState[ 1 ]( true ) }
```

Comment:
While it should technically work, we can array destructuring for readability as we do everywhere else.